engine/menus/naming_screen.asm

```diff
@@ -323,13 +329,27 @@ DisplayNamingScreen:
 	ld [wTopMenuItemX], a
 	jp EraseMenuCursor
 
+IF DEF(_ENGLISH)
 LoadEDTile:
 	ld de, ED_Tile
 	ld hl, vFont tile $70
 	ld bc, (ED_TileEnd - ED_Tile) / $8
 	; to fix the graphical bug on poor emulators
 	;lb bc, BANK(ED_Tile), (ED_TileEnd - ED_Tile) / $8
 	jp CopyVideoDataDouble
+ENDC
+
+IF DEF(_GERMAN) ; all european builds?
+LoadEDTile:
+; code from the english build results in a corrupted ED tile
```

Comment:
I don't think this comment is accurate. The ED tile is only corrupt on bad emulators that do not correctly mimic MBC behavior. AFAIK, it's never corrupt on real hardware. I am curious why they chose to rewrite this bit though.

Reply:
I couldn't test it on real hardware but I did test it on BGB. The US ROM displays the ED tile just fine but the german ROM has a garbled mess in place of the ED tile. Rewriting the function fixes this.